indico/htdocs/sass/modules/abstracts/_abstract_page.scss

```diff
@@ -206,6 +198,15 @@
 .abstract-review-badges {
     flex-grow: 1;
     color: $gray;
+
+    .js-ratings-toggle {
```

Comment:
It's the only class I have in the element. Would you artificially add another one? I think the idea with `js-*` classes is to make clear the class is needed for JS to trigger something.